includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -433,8 +435,33 @@ public function getData() {
 		return $data;
 	}
 
+	private function getSitenameData() {
+		$value = $this->wg->Sitename;
+
+		if ( $wgSitenameForComscore = WikiFactory::getVarValueByName( 'wgSitenameForComscore', $this->wikiId ) ) {
```

Comment:
This variable name is already used in global scope. It means that it's really easy to get into a problem where we get variable names conflict. Can we rename it to reflect fact that this variable is not related to current wiki?